packages/playwright-core/src/cli/program.ts

```diff
@@ -199,6 +199,29 @@ Examples:
   - $ install chrome firefox
     Install custom browsers, supports ${suggestedBrowsersToInstall()}.`);
 
+program
+    .command('list')
+    .description('List browsers used by this installation of Playwright')
+    .option('--all', 'List all browsers used by any Playwright installation.')
+    .action(async (options: { all?: boolean }) => {
+      await registry.list(!!options.all).then(browsersInfo => {
+        for (const info of browsersInfo) {
+          const whichInstanceLog = ` (${info.currentInstance ? 'CURRENT' : 'OTHER'})`;
```

Comment:
I kinda find `CURRENT/OTHER` being very much in my face. Can we tone it down somehow? Perhaps not show "other" at all, because we print the target directory anyway, and only mark current as `(current)` when `--all` is not passed?